src/views/Users/CreateUser/index.ts

```diff
@@ -0,0 +1,3 @@
+import CreateUser from './CreateUser';
+
+export default CreateUser;
```

Comment:
Either use the index.tsx or CreateUser.tsx. No need to maintain both everywhere in the project

Reply:
done